manifests/init.pp

```diff
@@ -17,6 +17,7 @@
   Optional[String]                                $cron_monthday = '*',
   Optional[String]                                $log_path      = undef,
   Enum['DEBUG', 'INFO', 'WARN', 'ERROR', 'FATAL'] $log_level     = 'WARN',
+  Enum['daily', 'weekly', 'monthly'] $log_schedule               = 'daily',
```

Comment:
Nit: Can we make the enum vals all caps? Just for consistency.
Nit2: Alignment of the variable - can we align the var name ($log_schedule to match up with the other variables? 
Can we also default it to the default value of the documentation? (I think it was 0? Or the value that disables the rotation?)

Reply:
Done!